index.html

```diff
@@ -29,14 +38,171 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
                     <li><a href="https://github.com/josman227"><i class="fa-sharp fa-solid fa-m"></i></a></li>
                     <li><a href="https://github.com/josman227"><i class="fa-brands fa-github"></i></a></li>
                     <li><a href="https://github.com/josman227"><i class="fa-brands fa-angellist"></i></a></li>
-        
+
                 </ul>
             </div>
+        </div>
             
         </div>
         
     </section>
+
+<section class="cards">
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
```

Comment:
-  [ ] Kindly consider removing the card title from the `div`. The `h2` tag is enough to render the title. This will help remove unnecessary code nesting and redundancy.